lib/puppet_x/puppetlabs/strings/yard/templates/default/hostclass/setup.rb

```diff
@@ -3,13 +3,6 @@
 def init
   super
   sections.push :subclasses
-
-  @template_helper = TemplateHelper.new
-  @template_helper.check_parameters_match_docs object
-  params = object.parameters.map { |param| param.first }
-  param_tags = object.tags.find_all{ |tag| tag.tag_name == "param"}
-  param_details = @template_helper.extract_param_details(params, param_tags) unless params.nil?
-  @template_helper.check_types_match_docs object, param_details
```

Comment:
So did we have this functionality in two places causing the duplication? All I see is this removal and then the spec updates, so I'm guessing this was already being handled somewhere else?